Assignment3Looping.java

```diff
@@ -0,0 +1,36 @@
+package assignment3looping;
+
+import java.util.Scanner;
+
+public class Assignment3Looping {
+
+    public static void main(String[] args) {
+        String word = "";
+        
+        do{
+            String backwardsWord = "";
+            print("Enter a word or QUIT to exit.\n");
+            

```

Comment:
I ran this through Netbeans.  It looked great.  I liked that you used a loop to let the user keep entering words or quit the program.